### PR TITLE
Fix: Resize Handler Console Error Fix (fixes #316)

### LINKED ISSUE #316: Console error while using resizeHandler
**What part of Carbon does this bug correlate to?**
General graph logic:

* [x]  Graph attributes (axes, legends, regions...etc.)
* [ ]  Other (please specify): 

Specific graph logic:

* [ ]  Bar graph
* [ ]  Bubble graph
* [ ]  Gantt chart
* [x]  Line graph
* [x]  Paired result
* [ ]  Pie chart
* [ ]  Scatter plot
* [ ]  Timeline

**Describe the bug**
There is a console error in our advanced-graphing application when we use resizeHandler().

**Expected behavior**
Graphs should be resized without any console error in the browser.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by JananiGunasekaran**
**What is the purpose of this pull request? (Check any that's applicable)**

-   [ ] Feature
-   [x] Bug fix
-   [ ] Dependency updates
-   [ ] Documentation updates
-   [ ] Build related changes
-   [ ] Changes an existing functionality
-   [ ] Other, please explain:

**Does this introduce a breaking change?**

-   [ ] Yes
-   [x] No

**What changes did you make? (Give an overview)**
Check if the graphContainer is present and then resize the graph.

**Add any screenshots**
Before
image
After
No console errors in the browser.

### CODE REVIEW

**Comment by sdadn** [comment]
@JananiGunasekaran Hi, Carbon Graphs is currently undergoing migration to a new repository: 
https://github.com/cerner/terra-graphs 

Can you create the PR in that repo after the migration is complete?

File: src/main/js/controls/Graph/Graph.js [hunk 1]
```diff
@@ -320,10 +320,13 @@ class Graph extends Construct {
...
-        scaleGraph(this.scale, this.config);
-        translateGraph(this);
-        this.content.forEach((control) => control.resize(this));
+        // Check if graphContainer is present and then resize the graph
+        if (this.graphContainer) {
+            setCanvasWidth(this.graphContainer, this.config);
+            scaleGraph(this.scale, this.config);
+            translateGraph(this);
+            this.content.forEach((control) => control.resize(this));
+        }
```
Comment (Veera81) [inline src/main/js/controls/Graph/Graph.js:329]: No unit tests for this change?

Comment (JananiGunasekaran) [reply src/main/js/controls/Graph/Graph.js:329]: Added unit test [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/f7635b1fd30a83f25a8a30fc27a7829a70e4e0f1).

**Comment by JananiGunasekaran** [comment]
> @JananiGunasekaran Hi, Carbon Graphs is currently undergoing migration to a new repository:
> https://github.com/cerner/terra-graphs
> 
> Can you create the PR in that repo after the migration is complete?

Sure. Can you please let me know when the migration will be done?

**Comment by sdadn** [comment]
Hopefully sometime next week.

**Comment by sdadn** [comment]
Migration is now complete.

**Pull request closed without merging**